cmd/kubectl-delivery/Dockerfile

```diff
@@ -1,14 +1,17 @@
-FROM alpine:3.10 AS build
+FROM golang:1.13.6 AS build
 
-# Install kubectl.
+WORKDIR /go/src/github.com/kubeflow/mpi-operator/
+COPY . /go/src/github.com/kubeflow/mpi-operator/
+RUN CGO_ENABLED=0 GOOS=linux go build -ldflags "-w" -o /bin/kubectl-delivery github.com/kubeflow/mpi-operator/cmd/kubectl-delivery
+# Install kubectl
 ENV K8S_VERSION v1.15.0
-RUN apk add --no-cache wget
+RUN apt-get install wget
 RUN wget -q https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl
 RUN chmod +x ./kubectl
 RUN mv ./kubectl /bin/kubectl
 
-FROM alpine:3.10
+FROM gcr.io/distroless/base-debian10
```

Comment:
`gcr.io/distroless/static-debian10` would be a better option since the code has been compiled with `CGO_ENABLED=0`. Could you please give it a try?

Reply:
```
# docker pull gcr.io/distroless/static-debian10
Using default tag: latest
Error response from daemon: manifest for gcr.io/distroless/static-debian10:latest not found: manifest unknown: Failed to fetch "latest" from request "/v2/distroless/static-debian10/manifests/latest".
```

Reply:
@carmark this is a known issue in GCR. Could you use the latest available tag instead?

For example:
https://console.cloud.google.com/gcr/images/distroless/GLOBAL/static-debian10@sha256:8fcd06c0c450a2354f1345d02d2713ceeb4e8ada98caf89e1e9f38f2a6788973/details?tab=info

Reply:
fixed by https://github.com/kubeflow/mpi-operator/pull/196/commits/ec6629a9373d87d982e9877e3efc15c1fd1e67b0